plugins/vinadapter/vinadapter.go

```diff
@@ -167,7 +167,7 @@ func generateVIN() (vin []byte) {
 	rand.Seed(time.Now().UnixNano())
 
 	for i := range vin {
-		vin[i] = vinSymbols[rand.Intn(len(vinSymbols))]
+		vin[i] = vinSymbols[rand.Intn(len(vinSymbols))] // nolint:gosec // test implementation
```

Comment:
What's wrong here?

Reply:
lint want to use crypto/rand. The random in this case not so critical.